hotsos/defs/scenarios/openstack/nova/bugs/lp2012284.yaml

```diff
@@ -0,0 +1,18 @@
+checks:
+  has_2012284:
+    input:
+      path: 'var/log/nova/nova-compute.log'
+    expr: ".+Failure prepping block device: gi.repository.GLib.GError: g-io-error-quark: Error opening directory '/usr/share/osinfo': Permission denied"
+    hint: 'osinfo'
+conclusions:
+  lp2012284:
+    decision: has_2012284
+    raises:
+      type: LaunchpadBug
+      bug-id: 2012284
+      message: >-
+        known nova compute issue caused when package gir1.2-libosinfo-1.0 is installed
+        and apparmor is set to enforced. To fix the issue, the offending package needs to
```

Comment:
I think it's actually "enforce" (without the d), at least according to the bug

Reply:
done